server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerAddHandler.java

```diff
@@ -171,8 +171,19 @@ static void installRuntimeServices(OperationContext context, ModelNode operation
                     }
                     authorizationConfig.getRoles().put(roleName, permissions);
                 }
+            }
+        }
 
+        if (model.hasDefined(ModelKeys.MODULES) && model.get(ModelKeys.MODULES).hasDefined(ModelKeys.MODULES_NAME)) {
+            ModelNode modulesModel = model.get(ModelKeys.MODULES, ModelKeys.MODULES_NAME);
+            LinkedList<ModuleIdentifier> modules = new LinkedList<>();
```

Comment:
Any reason this is a `LinkedList`? We could just initialize this as an `ArrayList` with size equal to the size of the list returned from `asList`.

Reply:
no particular reason. suggestion applied.